app/views/bikes/new.html.haml

```diff
@@ -1,62 +1,43 @@
-%a{ class: "btn btn-default btn-lg", href: root_path}
-  %span{ class:"icon-home"}
+%a{ class: "btn btn-default", href: root_path}
+  %span{ class:"fa fa-home"}
 %h2 Add Bike
 
 %p
-  .control-group
-    .controls
-      %input{id: "shop_id", placeholder: "Shop ID", type: "number", min:0, class: "input-lg" }
-      .help-block
-  .control-group
-    .controls
-      = select_tag(:bike_brand_id, options_for_select(@brands))
-      .help-block
-  .control-group
-    .controls
-      %input{id: "model", placeholder: "Model", type: "text", class: "input-lg" }
-      .help-block
-  .control-group
-    .controls
-      %input{id: "serial_number", placeholder: "Serial Number", type: "text", class: "input-lg" }
-      .help-block
-  .control-group
-    .controls
-      .btn-group{ "data-toggle" => "buttons-radio"}
-        %label{ class: "btn btn-default"}
-          %input{ type: "radio", name: "bike_style", value: 3} RD
-        %label{ class: "btn btn-default"}
-          %input{ type: "radio", name: "bike_style", value: 1} MTN
-        %label{ class: "btn btn-default"}
-          %input{ type: "radio", name: "bike_style", value: 2} HYB
-        %label{ class: "btn btn-default"}
-          %input{ type: "radio", name: "bike_style", value: 4} OTHER
+  .form-group
+    %input{id: "shop_id", placeholder: "Shop ID", type: "number", min:0, class: "form-control" }
+    .help-block
+  .form-group
+    = select_tag(:bike_brand_id, options_for_select(@brands),class:"form-control")
+    .help-block
+  .form-group
+    %input{id: "model", placeholder: "Model", type: "text" ,class:"form-control"}
+    .help-block
+  .form-group
+    %input{id: "serial_number", placeholder: "Serial Number", type: "text" ,class:"form-control"}
+    .help-block
+  .form-group
+    %input{ type: "radio", name: "bike_style", value: 3} RD
```

Comment:
Could you make the radio buttons look closer to the way they used to?
<img width="245" alt="screen shot 2016-07-27 at 2 52 57 pm" src="https://cloud.githubusercontent.com/assets/384389/17193876/eb74235a-5409-11e6-83be-b6ebe1a18373.png">
in the new bootstrap:
<img width="323" alt="screen shot 2016-07-27 at 2 46 28 pm" src="https://cloud.githubusercontent.com/assets/384389/17193701/18464620-5409-11e6-8042-634c05215eea.png">
 
instead of your current:

<img width="229" alt="screen shot 2016-07-27 at 2 48 32 pm" src="https://cloud.githubusercontent.com/assets/384389/17193729/36e86432-5409-11e6-8da1-b2aa930c4e23.png">